website/docs/examples/gps_client/README.md

```diff
@@ -0,0 +1,114 @@
+---
+id: gps-client
+title: GPS service client Example
+---
+
+This [example](https://github.com/farm-ng/farm-ng-amiga/blob/main/py/examples/gps_client/main.py)
+acts as a `GpsClient` in a standalone Python script.
```

Comment:
The `CameraClient` , `CanbusClient`, `GpsClient`, etc. classes no longer exist with the new API
```suggestion
acts as an `EventClient` to the GPS service in a standalone Python script.
```